ow_lander/action/Guardedmove.action

```diff
@@ -0,0 +1,10 @@
+# goal
+geometry_msgs/Point start     # arm start position
+geometry_msgs/Point normal    # vector normal to direction of approach
+float32 search_distance       # How deep along normal vector the scoop moves
+---
+# result
```

Comment:
Anjan, please let me know when the ground-found success flag is added here.  Limited testing of the reference mission plans until then, as they rely on ground found before doing more arm ops.

Reply:
Actually, I'm faking ground found for now, so not blocked in testing the reference mission.